src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int FIRST_PART = 0;
+    private static final int SECOND_PART = 1;
+    private static final int NULL_AMOUNT = 0;
+    private static final String SEPARATOR = ",";
+    private static final String SUPPLY_OPERATION = "s";
+    private static final String BUY_OPERATION = "b";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] result = calculateReport(fromFileName);
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+            writer.write("supply," + result[0] + System.lineSeparator()
+                    + "buy," + result[1] + System.lineSeparator()
+                    + "result," + result[2]);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to the file " + toFileName, e);
+        }
```

Comment:
this method combines IO and business logic, what violates SRP principle. feel free to use this template: 
```suggestion
        String dataFromFile = readFile(fromFileName);
        String report = createReport(dataFromFile);
        writeReportToFile(report, toFileName);
```